sdk/monitor/Azure.Monitor.OpenTelemetry/tests/Examples.AspNetCore/Examples.AspNetCore.csproj

```
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>net7.0</TargetFrameworks>
```

Try this to unblock the build:

```suggestion
    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```